ledger/tests/shred.rs

```diff
@@ -226,17 +226,20 @@ fn setup_different_sized_fec_blocks(
     let reed_solomon_cache = ReedSolomonCache::default();
     for i in 0..2 {
         let is_last = i == 1;
-        let (data_shreds, coding_shreds) = shredder.entries_to_shreds(
-            &keypair,
-            &entries,
-            is_last,
-            chained_merkle_root,
-            next_shred_index,
-            next_code_index,
-            true, // merkle_variant
-            &reed_solomon_cache,
-            &mut ProcessShredsStats::default(),
-        );
+        let shreds: Vec<Shred> = shredder
+            .make_merkle_shreds_from_entries(
+                &keypair,
+                &entries,
+                is_last,
+                chained_merkle_root,
+                next_shred_index,
+                next_code_index,
+                &reed_solomon_cache,
+                &mut ProcessShredsStats::default(),
+            )
+            .collect();
```

Comment:
You should prefer to partition first then collect, this does 2x the allocations. does not matter for a test but to be avoided in prod code.